schemas/telemetry/main/main.4.schema.json

```diff
@@ -4,7 +4,23 @@
   "mozPipelineMetadata": {
     "bq_dataset_family": "telemetry",
     "bq_metadata_format": "telemetry",
-    "bq_table": "main_v4"
+    "bq_table": "main_v4",
+    "split_config": {
+      "preserve_original": true,
```

Comment:
Is the plan to eventually set this to `false`?

Reply:
yes, once all consumers have been migrated to downstream pings